resources/language/en.ini

```diff
@@ -79,7 +90,13 @@ buttons.delete    = "Delete"
 buttons.select_file    = "Select File"
 buttons.change    = "Change"
 buttons.remove    = "Remove"
-buttons.upload    = "Upload"
+buttons.upload    =
```

Comment:
I think you missed the upload string

Reply:
This was an accident. I'll add it again.

Reply:
Fixed